### PR TITLE
[activemq] Add ECS fields

### PR DESCRIPTION
**Pull request by kush-elastic**
- Enhancement

## What does this PR do?
This pull request is raised to add some ECS fields to ActiveMQ package which could be useful for user experience. Following changes are made to enhance package.

- Added event.category
- Added event.kind
- Added event.type
- Updated ecs.yml with all the fields
- Updated version to 0.4.3 in manifest.yml

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## How to test this PR locally

- Clone integrations repo.
- Install elastic-package locally.
- Start elastic stack using elastic-package.
- Move to integrations/packages/activemq directory.
- Run the following command to run tests.

### CODE REVIEW

**Review by yug-elastic — APPROVED** [review]

File: packages/activemq/data_stream/queue/elasticsearch/ingest_pipeline/default.yml [hunk 1]
```diff
@@ -4,6 +4,21 @@ processors:
   - set:
       field: ecs.version
       value: 8.2.0
+  - set:
+      field: event.category
+      value: [web]
```
Comment (ishleenk17) [inline packages/activemq/data_stream/queue/elasticsearch/ingest_pipeline/default.yml:9]: Why do we need a [ ] for the value ? Same applies for all datastreams using event.category

Comment (yug-elastic) [reply packages/activemq/data_stream/queue/elasticsearch/ingest_pipeline/default.yml:9]: We referred to the note in the description of the field and have used the brackets based on that.

> Note: this field should contain an array of values.

https://www.elastic.co/guide/en/ecs/current/ecs-event.html#field-event-category

We have had a [similar discussion](https://github.com/elastic/integrations/pull/3666#discussion_r926619857) with @ritalwar in the past.

Comment (ishleenk17) [reply packages/activemq/data_stream/queue/elasticsearch/ingest_pipeline/default.yml:9]: If we keep this in mind then even event.type expects array values. 
Hence, if for single value you are putting a [ ], do that for event.type as well. 

Else, if you have single value for now, you can skip this usage and have it when/if more values come in.

Comment (kush-elastic) [reply packages/activemq/data_stream/queue/elasticsearch/ingest_pipeline/default.yml:9]: sure

Comment (kush-elastic) [reply packages/activemq/data_stream/queue/elasticsearch/ingest_pipeline/default.yml:9]: done

**Review by ishleenk17 — APPROVED** [review]
Looks good!

**Review by ishleenk17 — APPROVED** [review]
Looks good!

**Pull request merged**